prober/prober_test.go

```diff
@@ -0,0 +1,293 @@
+// Copyright (c) 2022 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package prober
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strings"
+	"sync"
+	"testing"
+	"time"
+
+	"tailscale.com/syncs"
+	"tailscale.com/tstest"
+)
+
+const (
+	probeInterval        = 10 * time.Second // So expvars that are integer numbers of seconds change
```

Comment:
8 seconds? 10 won't divide nicely by 4 :)

Reply:
It's all nanoseconds under the hood, so it should be fine, but sure :)